### DISCUSSION TITLE
Unexpected Behavior - Reset Auth with Azure AD Authentication

### DISCUSSION THREAD

**Discussion by jpmckearin** [datalust/seq-tickets#1560] [Help]
We are running a Seq instance (datalust/seq:2022.1.7449) on a Kubernetes cluster on AKS using a PersistentVolumeClaim mounted in the `/data` directory. I set up Active Directory authentication using a client secret from our Azure AD instance. This was working great until the client secret  expired. There is no way to log in using basic auth to update this secret from the UI. Therefore, according to the [documentation](https://docs.datalust.co/docs/recovering-from-broken-authentication-settings#resetting-the-authentication-provider-and-unlocking-the-user-account), I need to run the `auth reset` command using the seq cli. Since I don’t have host shell access, I created a `reset-auth.sh` file in the `/data/Init` directory and restarted the pod. It would error unless I specified an existing user so I did the following:
 
```sh
#!/bin/bash
./bin/seqsvr auth reset --storage=/data -u knownuser@domain.com -p admin
```

The file ran successfully as it was renamed to `reset-auth.sh.done` appropriately. After ensuring that I reset cache and hard reload Seq is still attempting to log in using the Active Directory credentials. I am at a loss at this point. Any assistance would be much appreciated.
 
Thanks!

**Comment by nblumhardt** [comment]
Sorry, I see your other request is not an exact duplicate; re-typing below.

Our best bet is that you're running Seq 2021.4, while `seq auth reset` is the 2022.1 syntax. Without the -e option, it's possible that the seqsvr call is failing but the script itself is not.

For Seq 2021.4, the init script contents would need to be:

```
#!/bin/bash
set -e
seqsvr auth --basic -u knownuser@domain.com -p admin
```

You shouldn't need to qualify the path to seqsvr, nor specify the --storage argument.
